src/components/FetchRouteList.tsx

```diff
@@ -0,0 +1,70 @@
+import { Badge, Text } from "@fluentui/react-components";
+import { Card } from "@fluentui/react-components/unstable";
+import { useEffect, useState } from "react";
+import { Link } from "react-router-dom";
+
+import { RoutesXml } from "../data/etaXml";
+import RawDisplay from "./RawDisplay";
+import { FetchXMLWithCancelToken } from "./fetchUtils";
+
+const parseRouteTitle = (input: string) => {
+  const routeTitleRegex = /\d+-/;
+  if (routeTitleRegex.test(input)) {
+    return input.replace(routeTitleRegex, "");
+  } else {
+    return input;
+  }
+};
+
+export function RoutesInfo() {
+  const controller = new AbortController();
+  const [data, setData] = useState<RoutesXml>();
```

Comment:
I made that mistake in other component too :sorry:

shd we rename data to `routeXmlData`, `data` seems a bit unclear.